CMakeLists.txt

```diff
@@ -287,6 +291,16 @@ if (DOWNLOAD_OBJECTS)
             file(REMOVE \$ENV{DESTDIR}/${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/object/objects.zip)\n\
         endif ()")
 endif ()
+if (DOWNLOAD_REPLAYS)
+    install(CODE
+        "if (EXISTS \$ENV{DESTDIR}${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/testdata/replays/)\n\
```

Comment:
Oh, this is no good. It should inspect a path that exists _before_ we issue an `install` target, like this: https://github.com/OpenRCT2/OpenRCT2/blob/18e35c1c98b6a084f58264862d05cd56399e2f7b/CMakeLists.txt#L286

Otherwise I can't inject the replays into the build system and it will fail to build.

Reply:
The launchpad thing doesn't need the replays though as it just needs to build, not test. Can't you disable the download?

Reply:
That's a fair point, the replays should only be downloaded when building with tests. Thanks for pointing that out